src/extension.js

```diff
@@ -443,13 +390,13 @@ function init() {
 
 
 function enable() {
-    serviceIndicator = new ServiceIndicator();
+    featureIndicator = new FeatureIndicator();
```

Comment:
```suggestion
    serviceIndicator = new ServiceIndicator();
```

Renaming for consistency (ServiceIndicator, ServiceToggle).